src/components/Landing/LastPosts/LastPosts.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link, navigate } from 'gatsby';
+import { GatsbyImage } from 'gatsby-plugin-image';
+import styles from './LastPosts.module.scss';
+
+const LastPosts = ({ lastPosts }) => (
+    <div className={styles['container']}>
+      {lastPosts.map((post, i) => (
+        <div key={i} className={styles['card']}>
+          <p className={styles['category']}>{post.node.frontmatter.category.toUpperCase()}</p>
+          <GatsbyImage
+            image={post.node.frontmatter.socialImage.childImageSharp.gatsbyImageData}
+            alt={post.node.frontmatter.title}
+            onClick={() => navigate(post.node.fields.slug)}/>
```

Comment:
You're using the `navigate()` function to go to the post. Is there a reaso why you're not using `Link`? How to `navigate()` work?